backend/package.json

```diff
@@ -4,10 +4,12 @@
   "license": "MIT",
   "scripts": {
     "start": "NODE_ENV=prod node server.js",
-    "startDatabase": "docker run -d --name buckets_db -p 27017:27017 -p 28017:28017 -e MONGODB_USER=\"username\" -e MONGODB_DATABASE=\"buckets_test\" -e MONGODB_PASS=\"password\" -e AUTH=no tutum/mongodb",
+    "start_db": "docker run -d --name buckets_db -p 27017:27017 -p 28017:28017 -e MONGODB_USER=\"username\" -e MONGODB_DATABASE=\"buckets_test\" -e MONGODB_PASS=\"password\" -e AUTH=no tutum/mongodb",
```

Comment:
What is my connection string for this?
<img width="571" alt="screen shot 2017-08-06 at 2 44 39 pm" src="https://user-images.githubusercontent.com/4207065/29006456-d5960a84-7ab5-11e7-8f3a-02615882c577.png">